pack/tde/tde.json

```diff
@@ -639,16 +656,17 @@
 		"quantity": 1,
 		"skill_agility": 1,
 		"skill_combat": 1,
-		"text": "Bonded (Miss Doyle). Fast.\n<b>Forced</b> - After Augur enters play: Discard Hope and Zeal.\n[action] If Augur is ready, exhaust or discard her: <b>Investigate.</b> Investigate with a base [intellect] value of 5. (If you discard Augur, this test is automatically successful. Then, you may shuffle Augur into your deck to put Hope or Zeal into play from your discard pile.)",
+		"text": "Bonded (Miss Doyle). Fast.\n<b>Forced</b> - After Augur enters play: Discard Hope and Zeal.\n[action] If Augur is ready, exhaust or discard her: <b>Investigate.</b> Investigate with a base [intellect] value of 5. (If you discarded Augur, this test is automatically successful. Then, you may shuffle Augur into your deck to put Hope or Zeal into play from your discard pile.)",
 		"traits": "Ally. Creature. Dreamlands.",
 		"type_code": "asset"
 	},
 	{
 		"code": "06034",
-		"cost": null,
+		"cost": -2,
```

Comment:
I looked into this and I don't think we have ever set cost to -2 for the - vs X case.  It might work, but it might not.  @Kamalisk thoughts?  

It does seem like the old differentiation on costs is gone, half of them are now X and half are -

Reply:
Yeah, I'd like to know too!

Reply:
at least for shroud and enemy fight/evade -2 is shown as X. I don't think we ever did that for player cards and left it at null.

Reply:
Anything less than 0 is shown as X and anything null is shown as -